setup.py

```diff
@@ -10,9 +10,9 @@
     "gspread-pandas>=3.2.2",
     "gspread-dataframe>=3.3.0",
     "gspread-formatting>=1.1.2",
-    "duckdb>=0.7.1",
+    "duckdb>=0.8.2.dev4514",
```

Comment:
Why this particular dev version? Is that just the latest one?

Reply:
If we don't actually need something beyond 0.8.1, I would prefer to just use an official release rather than a dev one, as dev ones may disappear some day, or end up changing in functionality by the time they are actually released.

Reply:
+1

Reply:
Only `duckdb>=0.8.2` will support newest `pandas` version, however version `0.8.2` of `duckdb` is not released yet. 

To support Pandas 2 (Streamlit added a support for Pandas 2, so `gsheets-connection` should too) I have to use dev version of `duckdb`. 

We can do a follow up PR and release once official release `0.8.2` of `duckdb` is published or wait for `duckdb==0.8.2` release, what do you think @sfc-gh-jcarroll @sfc-gh-zblackwood ?

Reply:
Thank you for conversation, thanks to it a got a chance to thought it through, since the Pypi owners can remove dev package at any time, let's enforce Pandas <2 for now, until Duckdb 0.8.2 is officially released